entrypoint.sh

```diff
@@ -15,5 +15,13 @@ if [ "$1" == "install" ]; then
     exit
   fi
 else
-  exec kube-bench "$@"
+  if [ -n "$SCHEDULE" ]; then
+    echo "$SCHEDULE" "cd $PWD && date && kube-bench" "$@" | crontab -c . -
+    crond -c . -f
+  else
+    kube-bench "$@"
+    # If the SCHEDULE variable is not set, the container exits.
+    # If set, the container sleeps.
+    [ -n "${SCHEDULE+set}" ] && while :; do sleep 86400; done
```

Comment:
what is the purpose of `+set` here?

Reply:
The Dockerfile had to accommodate two ways of running. 1) As a container in a pod as part of a Kubernetes deployment/daemonset 2) as a plain docker container that runs directly on the host. If `SCHEDULE` is set, it is assumed to be in mode 1 where the container sleeps after a single run if there is no schedule. This would prevent the daemonset from restarting the pod with an exited container over and over if the intention is to just run kube-bench once and then delete the deployment. If the variable is not set, it is assumed to run as a standalone container and it exits after running kube-bench